docs/aws/cluster-parameters.md

```diff
@@ -15,3 +15,39 @@ clusterawsadm ami list
 ```
 
 For details, see [Pre-built Kubernetes AMIs](https://cluster-api-aws.sigs.k8s.io/topics/images/built-amis.html).
+
+## SSH access to cluster nodes
+
+In order to access the nodes using SSH protocol, several things should be configured:
+
+- Apropriate SSH keys are added in the region where you want to deploy the cluster
+- Bastion host enabled
+
+### SSH keys
+
+Only one SSH key is supported and it should be added in AWS preor creating
+the `Deployment` object name of the key then can be placed under `.spec.config.sshKeyName`
+
+For the bastion host key named **default** must be created, since setting key name
+for bastion host is not possible.
+
+To enable bastion you should add `.spec.config.bastion.enabled` option in the
```

Comment:
We aim to provide default values for all configurable options (it's acceptable if only keys are present). It also would be nice to adjust JSON schemas with the newly added params.

Reply:
I expanded all available bastion options and did some basic tests. values.schema.json updated as well